src/libstore/store-api.hh

```diff
@@ -695,6 +695,8 @@ public:
 
     const Stats & getStats();
 
+    StorePathSet exportReferences(const StorePathSet & storePaths, const StorePathSet & inputPaths);
```

Comment:
This needs a comment about what `exportReferences` does. Also the name is not ideal, since it doesn't really export anything... (Especially confusing since there is a `nix-store --export` command.)